src/Ledger/Ratify.lagda

```diff
@@ -410,22 +411,25 @@ The code in Figure~\ref{fig:defs:ratify-iii} defines yet more types required for
 {\small
 \begin{code}
 verifyPrev : (a : GovAction) → NeedsHash a → EnactState → Set
-verifyPrev NoConfidence           h es = let open EnactState es in h ≡ proj₂ cc
-verifyPrev (NewCommittee _ _ _)   h es = let open EnactState es in h ≡ proj₂ cc
-verifyPrev (NewConstitution _ _)  h es = let open EnactState es in h ≡ proj₂ constitution
-verifyPrev (TriggerHF _)          h es = let open EnactState es in h ≡ proj₂ pv
-verifyPrev (ChangePParams _)      h es = let open EnactState es in h ≡ proj₂ pparams
-verifyPrev (TreasuryWdrl _)       _ _  = ⊤
-verifyPrev Info                   _ _  = ⊤
+verifyPrev action h es = go where
```

Comment:
I see why you changed this, but this will be completely unreadable by non-Agda experts.